.github/CODEOWNERS

```diff
@@ -4,7 +4,20 @@
 /stdlib/       @modularml/mojo-standard-library
 
 # Documentation
-/docs/         @modularml/mojo-docs
+# Docs team: Everything except the unreleased changelog
```

Comment:
**Question** Since `CODEOWNERS` is effectively processed bottom to top and all of the changelog-specific changes are only for the stdlib right now (nothing else is open-source), would it instead work to add a rule

```
docs/changelog.md @modularml/mojo-standard-library
```

Reply:
Oh, that would be oh so much cleaner. 🙇